src/pages/AzurePreview.test.js

```diff
@@ -132,6 +132,12 @@ describe('AzurePreview', () => {
           await user.clear(screen.getByLabelText('Contact email address *'))
           await user.type(screen.getByLabelText('Contact email address *'), 'user@example.com')
 
+          // Assert
+          expect(isSubmitEnabled()).toBe(false)
```

Comment:
nit: (don't hold up PR for this)
I prefer to save intermediate stuff that we want to assert like this into a local var, then assert that var's value later in a unified // Assert section at the end of the test:

```
const initialSubmitEnabled = isSubmitEnabled()
// ....
// Assert
expect(initialSubmitEnabled).toBe(false)
```